### PR TITLE
Makefile cleaned up to accomodate glide

### PR DESCRIPTION
**Pull request by ramkrsna**
- This patch removes the need for godeps in the heketi project.
- The Makefile has been modified to get the vendor directory and then build the application.
- We could choose to not ship the vendor directory in future. 

Signed-off-by: Ramakrishna Reddy Yekulla <rreddy@redhat.com>

### CODE REVIEW

File: client/cli/go/Makefile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-#
+:#
```
Comment (obnoxxx) [inline client/cli/go/Makefile]: What's that ":" ? I think that is by accident and needs to be removed?...

File: client/cli/go/Makefile [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 # Based on http://chrismckenzie.io/post/deploying-with-golang/
 #
 
 .PHONY: version all run dist clean
-
```
Comment (obnoxxx) [inline client/cli/go/Makefile]: sneaked-in whitespace cleanup? ... well ok. ;-)

**Comment by obnoxxx** [comment]
@ramkrsna and note that while the patch looks good in general, the travis checks are still failing. i.e. the changes are not sufficient yet.

**Comment by lpabon** [comment]
Replaced by https://github.com/heketi/heketi/pull/769

**Pull request closed without merging**